mpipartition/overload.py

```diff
@@ -64,7 +64,12 @@ def overload(
     """
     assert len(coord_keys) == partition.dimensions
     for i in range(partition.dimensions):
-        assert overload_length < 0.5 * partition.extent[i] * box_size
+        max_overload = partition.extent[i] * box_size
+        if partition.decomposition[i] == 2:
+            # can maximally overload have the local extent, otherwise would need
```

Comment:
```suggestion
        if partition.decomposition[i] <= 2:
            # can maximally overload half the local extent, otherwise would need
```

Reply:
@michaelbuehlmann I think this same limit applies to 1 too?

Reply:
I'm actually not sure at the moment what happens if there's only 1. I'll take a closer look – maybe we should add an assert that it's at least 2 for now?

Reply:
Yeah that seems safe for now, though it is a bit unfortunate.